### ISSUE TITLE
Implement advanced verification of SD-JWT

### ISSUE THREAD

**Issue by francis-pouatcha** [adorsys/keycloak-oid4vc#17]
### Description

Spec: https://drafts.oauth.net/oauth-selective-disclosure-jwt/draft-ietf-oauth-selective-disclosure-jwt.html#section-8.1
Spec: https://drafts.oauth.net/oauth-selective-disclosure-jwt/draft-ietf-oauth-selective-disclosure-jwt.html#section-8.3
Spec: https://drafts.oauth.net/oauth-selective-disclosure-jwt/draft-ietf-oauth-selective-disclosure-jwt.html#section-11.1
Spec: https://drafts.oauth.net/oauth-selective-disclosure-jwt/draft-ietf-oauth-selective-disclosure-jwt.html#section-11.2

### Dependency
Depends on: #9 

### Before You Start
Have a look at [how to contribute](https://github.com/adorsys/keycloak-oid4vc/discussions/19)

**Comment by francis-pouatcha** [comment]
Integrity check - can be performed in consuming module:
- check vc is signed.
- check no ghost digest
- If bound, check keyBindingJWT

**Comment by IngridPuppet** [comment]
Issue https://github.com/adorsys/keycloak-oid4vc/issues/9 (on which this task depends) was done in https://github.com/adorsys/sd-jwt and has not been merged into https://github.com/adorsys/keycloak-oid4vc. Should I keep working from https://github.com/adorsys/sd-jwt?

**Comment by francis-pouatcha** [comment]
Yes please keep working on https://github.com/adorsys/sd-jwt. Result will be taken over to https://github.com/adorsys/keycloak-oid4vc.
